src/compiler/clang.rs

```diff
@@ -38,9 +38,9 @@ use crate::errors::*;
 /// A struct on which to implement `CCompilerImpl`.
 #[derive(Clone, Debug)]
 pub struct Clang {
-    /// true iff this is clang++.
+    /// true if this is clang++.
```

Comment:
why this change?
iff means `If and only if`

Reply:
Just seems odd, as "if" is totally sufficient in this case (otherwise we would write `iff true {`, wouldn’t we?). And yes, I wasn’t aware ;)